### PR TITLE
Start the inventory section with 3 first recommendations

### PR DESCRIPTION
**Pull request by ericzolf**
* Identify your Single Source(s) of Truth and use it/them in your inventory
* Differentiate clearly between As-Is and To-Be information
* Define your inventory as structured directory instead of single file

### CODE REVIEW

File: inventories/README.adoc [hunk 1]
```diff
@@ -1 +1,159 @@
...
+* discovered information grabbed from the existing environment, this is the "As-Is" information.
+* managed information entered in a tool, expressing the state to be reached, hence the "To-Be" information.
++
+In general, the focus of an inventory is on the managed information because it represents the desired state you want to reach with your automation. This said, some discovered information is required for the automation to work.
+
+Rationale::
+Mixing up these two kind of information can lead to your automation taking the wrong course of action by thinking that the current situation is aligned with the desired state.
+That can make your automation go awry and your automation engineers confused.
+There is a reason why Ansible makes the difference between "facts" (As-Is) and "variables" (To-Be), and so should you.
+Automation is at the end to make sure that the As-Is situation complies to the To-Be description.
```
Comment (mophahr) [inline inventories/README.adoc:43]: How about:

```suggestion
In the end, automation is making sure that the As-Is situation complies to the To-Be description.
```

**Review by mophahr — CHANGES_REQUESTED** [review]
Looks very good. I have left suggestions but other than that, :+1: from my side.

File: inventories/inventory_example/host_vars/host1.example.com/ansible.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+# ansible specific variables
+ansible_host: 192.168.0.1
```
[the remark this answers is not in the document]
Comment (ericzolf) [reply inventories/inventory_example/host_vars/host1.example.com/ansible.yml:3]: done.

File: inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+# variables for the small_caps_letter role
+# font of the small caps letter (default: Sans)
+small_caps_letter_font: "Oxygen Mono"
```
[the remark this answers is not in the document]
Comment (ericzolf) [reply inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml:4]: Why? We don't have yet a recommendation for "strings" but I would have suggested something like: "put variable strings in quotes unless they represent a set of pre-defined values made of a single word in a module, e.g. absent/present".

File: inventories/README.adoc (hunk 1 above)
Comment (ericzolf) [reply inventories/README.adoc:43]: Your English is generally better than mine. Done.

File: inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (mophahr) [reply inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml:4]: If I remember correctly, you had no quotes in the other file(s).

**Review by mophahr — APPROVED** [review]

File: inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ericzolf) [reply inventories/inventory_example/group_vars/greek_letters/small_caps_letter.yml:4]: I admit, not one of the points where I keep best consistency... I'll improve :smirk:

**Comment by jscar-hawk** [comment]
Reviewed Approved at Automation CoP Meeting

**Pull request merged**